elastic/elastic-agent
# Log when `container-paths.yml` is loaded

## What does this PR do?

See title

## Why is it important?

It adds logs when the Elastic-Agent loads `container-paths.yml`

## Checklist


## Disruptive User Impact

None

## How to test this PR locally

Export the following environmet variables:
- `FLEET_ENROLL`
- `FLEET_URL`
- `FLEET_ENROLLMENT_TOKEN`

Run the container command as root
```
./elastic-agent container
```

Stop the Elastic-Agent and run the status command:
```
./elastic-agent status
```

You should see the logs:
```
2024/09/06 16:54:33 container path file '/usr/share/elastic-agent/state/container-paths.yml' found
2024/09/06 16:54:33 state Path: '/usr/share/elastic-agent/state', config path: '/usr/share/elastic-agent/state', logs path: '', socket path: 'unix:///usr/share/elastic-agent/state/data/Td8I7R-Zby36_zF_IOd9QVNlFblNEro3.sock'
```

~~## Related issues~~

## Questions to ask yourself

- How are we going to support this in production?
- How are we going to measure its adoption?
- How are we going to debug this?
- What are the metrics I should take care of?
- ...
